ANMN/burst_averaged_product/generate_nc_file_att

```diff
@@ -2,4 +2,4 @@
 author_email            = laurent.besnard@utas.edu.au
 author                  = Besnard, Laurent
 file_version            = Level 2 - Derived Products
-lineage                 = The data array in this file has been created by binning (averaging) raw burst data. Each array value is the arithmetic mean of a burst. Length and frequency of the underlying bursts are declared in the global attributes instrument_burst_duration and instrument_burst_interval respectively. Out-of-water data is excised before binning. Points flagged by IMOS QC as 4 are excluded from the burst average: this includes data values outside of a valid range, data depth outside of a valid range, and data values in a series of consecutive identical values whose length exceeds a maximum. For all parameters except those collected by optical sensor (TURB, CPHL, CHLR, CHLU, CHLF, FLU2), points flagged by IMOS QC as 3 are also removed. These flags are produced by the IMOS rate of change test: this is a test to detect spikes. This sort of test is not used to exclude any optical sensor data, because understanding of behaviour of these parameters at this time scale is still in its early stages. Burst means are calculated from remaining points. Any bursts with no remaining good points after exclusions are assigned FillValue (see variable attributes for value of FillValue).
+lineage                 = The data array in this file has been created by binning (averaging) raw burst data. Each array value is the arithmetic mean of a burst. Length and frequency of the underlying bursts are declared in the global attributes instrument_burst_duration and instrument_burst_interval respectively. Out-of-water data is excised before binning. Points flagged by IMOS QC as 3 and above are excluded from the burst average: this includes data values outside of a valid range, data depth outside of a valid range, and data values in a series of consecutive identical values whose length exceeds a maximum. These flags are produced by the IMOS rate of change test: this is a test to detect spikes. This sort of test is not used to exclude any optical sensor data, because understanding of behaviour of these parameters at this time scale is still in its early stages. Burst means are calculated from remaining points. Any bursts with no remaining good points after exclusions are assigned FillValue (see variable attributes for value of FillValue).
```

Comment:
I didn't have a look at this so far. Sorry but this will need to be removed:

>: this includes data values outside of a valid range, data depth outside of a valid range, and data values in a series of consecutive identical values whose length exceeds a maximum. These flags are produced by the IMOS rate of change test: this is a test to detect spikes. This sort of test is not used to exclude any optical sensor data, because understanding of behaviour of these parameters at this time scale is still in its early stages.

Also, it wouldn't harm to say again here that the time stamp lays in the middle of the burst. Can you please check this is the case? Could also be `(tn - t1)/2` with `t1` the time of the first good data in burst and `tn` the time of the last good data in burst.